### ISSUE TITLE
Long / Lat address determination is at best wrong

### ISSUE THREAD

**Issue by herrold** [5calls/5calls#344]
I am near: 40 N, 83 W

one assumes the IP owner (ATT)'s Texas address is being used

   http://gallery.herrold.com/stuff/5calls-PII-in-flight.png

**Comment by nickoneill** [comment]
This is our fallback mechanism for location. We attempt browser geolocation first (pretty accurate), then fall back to IP location if that's unavailable/denied.

And, in a case like yours where IP provides a poor location, you can always tap the location button and set it manually to an address or cross streets.

**Comment by herrold** [comment]
my thought is:  when something in the grid: 
> 37.7, 97.8

it is suspect, and similar to 
> https://github.com/5calls/android/issues/59

add an exception handler to treat it as a non-useful value

**Comment by nickoneill** [comment]
Right! This is that default "center of the US" point that is returned when that's the most it can say about an IP location. We should definitely capture this as treat it as an invalid location. I'll rework the issue to be more specific.
